src/lib.rs

```diff
@@ -260,26 +199,7 @@ fn addr_to_tuple(input: &str) -> (String, u16) {
 #[inline]
 fn span_status(http_status: tide::StatusCode) -> StatusCode {
     match http_status as u16 {
-        100..=399 => StatusCode::OK,
```

Comment:
The opentracing library changed last month removing these status codes to conform with the spec which only has Ok, Error, or unknown.